infrastructure/src/main/java/org/corfudb/infrastructure/ClusterMetrics.java

```diff
@@ -2,24 +2,31 @@
 
 import lombok.Getter;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.util.Map;
 
 /**
  * ClusterMetrics:
  * Contains a list of all records accumulated by
  * the polling component from all other nodes.
+ * This table is maintained in memory with the management server.
```

Comment:
Sorry, it is a bit confusing. We have ClusterMetrics, ClusterView and clusterStatus. 
We also have have NodeView, NodeMetrics and NodeHealth.

Could we please choose names that are easier to distinguish?